internal/web/controllers/users-controller.go

```diff
@@ -114,6 +114,21 @@ func (c *UsersController) GetUserById(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(user)
 }
 
+func (c *UsersController) GetAdmins(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	users, serviceErr := c.service.GetUsers(nil)
```

Comment:
Seguindo o comentário do `service`:
```suggestion
	users, serviceErr := c.service.GetUsers(myTypes.Where{
		"role": map[string]string{"eq": "admin"},
	})
```